### PR TITLE
feat: support reading environment variables.

### PR DESCRIPTION
**Pull request by hugogu**
Trying to resolve issue: https://github.com/leonyork/xk6-output-timestream/issues/260

Tested in my local. Seems to be working well. output prints variables passing from env.

```log
(base) hugogu@HugodeMacBook-Pro load-test %  ./bin/k6 run -o timestream -e K6_TIMESTREAM_DATABASE_NAME='timestream_db1' \
  -e K6_TIMESTREAM_TABLE_NAME='load-test' - < ./scripts/max-vu.js

          /\      |‾‾| /‾‾/   /‾‾/   
     /\  /  \     |  |/  /   /  /    
    /  \/    \    |     (   /   ‾‾\  
   /          \   |  |\  \ |  (‾)  | 
  / __________ \  |__| \__\ \_____/ .io

  execution: local
     script: -
     output: Timestream (timestream_db1:load-test)
```

### CODE REVIEW

**Comment by leonyork** [comment]
I'm taking a look at the permissions errors - looks like it's not letting to write to packages in GHA. Let me change to use the github cache.

**Comment by hugogu** [comment]
> I'm taking a look at the permissions errors - looks like it's not letting to write to packages in GHA. Let me change to use the github cache.
> 
> In the meantime I think you're missing add the new dependency to `go.mod` (if you see in [grafana/xk6-output-influxdb@46b84ad](https://github.com/grafana/xk6-output-influxdb/commit/46b84adaaa3a7f3a0b15413ca5084ea2e025fac6) there's also the update to add the new dependency)

Thanks @leonyork. I have updated go.mod and amended last commit (as it hasn't been merged yet hopefully safe.) Please let me know if any further changes requires.

**Comment by hugogu** [comment]
@leonyork Is there anything I can do with the permission error please?

**Comment by leonyork** [comment]
@hugogu it looks like config_test.go will need to be updated with the change to the signature for GetConsolidatedConfig - see https://github.com/leonyork/xk6-output-timestream/actions/runs/7681376154/job/20934503275?pr=261#step:4:64

I'm just looking into why [this test](https://github.com/leonyork/xk6-output-timestream/actions/runs/7681376153/job/20934503266?pr=261) is failing - looks like it's not able to get credentials to run the integration tests.

**Comment by leonyork** [comment]
@hugogu it looks like config_test.go will need to be updated with the change to the signature for GetConsolidatedConfig - see https://github.com/leonyork/xk6-output-timestream/actions/runs/7686849645/job/20946192572?pr=261#step:4:64. Note that you should be able to replicate this locally by running `go test`.

I've fixed the permissions issue, and the integration tests are passing, so it should just be the unit tests left to sort!

Thank you for your patience.

File: config.go [hunk 1]
```diff
@@ -47,24 +47,24 @@ func parseJSON(data json.RawMessage) (Config, error) {
 // GetConsolidatedConfig combines {default config values + JSON config +
 // environment vars config values}, and returns the final result.
 func GetConsolidatedConfig(
-	jsonRawConf json.RawMessage,
+	jsonRawConf json.RawMessage, env map[string]string, url string,
```
Comment (leonyork) [inline config.go:50]:
```suggestion
	jsonRawConf json.RawMessage, env map[string]string,
```

I'd suggest removing the URL parameter from this line as it's not used. It looks like this came from https://github.com/grafana/xk6-output-influxdb/blob/main/pkg/influxdb/config.go#L116 where the influxdb location can be specified as a URI. I don't believe there's the same concept for Timestream, and to keep this PR focussed on just the environment parsing, I'd leave this out.

File: output.go [hunk 2]
```diff
@@ -51,7 +51,7 @@ type Output struct {
 }
 
 func New(params output.Params) (output.Output, error) {
-	extensionConfig, err := GetConsolidatedConfig(params.JSONConfig)
+	extensionConfig, err := GetConsolidatedConfig(params.JSONConfig, params.Environment, params.ConfigArgument)
```
Comment (leonyork) [inline output.go:54]:
```suggestion
	extensionConfig, err := GetConsolidatedConfig(params.JSONConfig, params.Environment)
```

I've suggested removing the URL parameter in 
https://github.com/leonyork/xk6-output-timestream/pull/261/files#r1468902395 so don't forget to remove here where we call it!

**Review by leonyork — CHANGES_REQUESTED** [review]

**Comment by leonyork** [comment]
Hi @hugogu! Do you have time to pull in the fixes above? I'm happy to take this on from here if that helps? (it would be nice to have another contributor though). Let me know what you think 😃

File: config.go (hunk 1 above)
Comment (hugogu) [reply config.go:50]: Ah, that's right. Let me remove it.

**Comment by hugogu** [comment]
Hi @leonyork Sorry for the late as I was on national holiday for the past a few weeks. Let me make the changes you suggested. Thanks for your patience.

**Review by leonyork — APPROVED** [review]

**Comment by leonyork** [comment]
Nice work on the PR and I hope you had a great holiday! I'll merge it in. Thank you 😃

**Pull request merged**